ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/command/CommandStaticTimeIT.scala

```diff
@@ -102,33 +161,38 @@ class CommandStaticTimeIT
     }.head
   }
 
-  private def timeSource(ctx: LedgerContext) =
-    ClientAdapter.serverStreaming(GetTimeRequest(ctx.ledgerId.unwrap), ctx.timeService.getTime)
+  private def timeSource =
+    ClientAdapter.serverStreaming(
+      GetTimeRequest(unwrappedLedgerId),
+      TimeServiceGrpc.stub(channel).getTime)
 
   "Command and Time Services" when {
 
     "ledger effective time is within acceptance window" should {
 
-      "commands should be accepted" in allFixtures { ctx =>
+      "commands should be accepted" in {
         for {
-          commandClient <- ctx.commandClient()
+          commandClient <- commandClient
```

Comment:
Please rename one of these. Preferably the function. Call it `newCommandClient` or something.

I also think it'd be nice if it had parentheses. It's not a pure function, exactly—it uses the materializer.

Reply:
Makes a lot of sense, thanks.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/3874/commits/2d27ff8655ae1e109a97201be518807c241f1cf5